### PR TITLE
Update issue management guidelines and labels

### PR DESCRIPTION
**Pull request by harendra-kumar**

### CODE REVIEW

**Comment by adithyaov** [comment]
What about pull requests? What are the additional labels and what labels can be shared by pull requests?

**Review by adithyaov — APPROVED** [review]
LGTM.

**Comment by harendra-kumar** [comment]
> What are the additional labels and what labels can be shared by pull requests?

I don't think we need labels for PRs. Pending PRs are fewer and usually short lived.

File: MAINTAINING.md [hunk 1]
```diff
@@ -142,49 +142,78 @@ Internal APIs:
...
+creating it.
+
+### Disposition
+
+The level-1 triaging of the issue determines the current disposition
+of the issue. If no change is required the issue must have one of the
+following labels:
 
+* invalid
 * question
```
Comment (rnjtranjan) [inline MAINTAINING.md:157]: question and discussion can be combined to "Need More Info"

File: MAINTAINING.md [hunk 2]
```diff
@@ -142,49 +142,78 @@ Internal APIs:
...
+
+* aspect:<aspect name>
+
+### Severity
+
+Optionally we can add a severity label to indicate how severe is the
+impact of the bug/issue, which may determine how important it is to fix it. By
+default the severity is normal, if it is high we put a label:
+
+* sev:high
```
Comment (pranaysashank) [inline MAINTAINING.md:188]: I don't see the need for a shorthand,  `severity:high` seems fine.

File: MAINTAINING.md [hunk 3]
```diff
@@ -142,49 +142,78 @@ Internal APIs:
...
+__RULE__: Any commit that may affect the end user in some way MUST have
+either a changelog entry OR MUST have an issue marked with one of the
+above labels OR both.
+
+### Priority
+
+By default the issues are normal priority. We use a label for high priority
+issues:
+
+* pri:high
```
Comment (pranaysashank) [inline MAINTAINING.md:207]: `priority:high` ?

**Review by pranaysashank — CHANGES_REQUESTED** [review]

File: MAINTAINING.md (hunk 1 above)
Comment (harendra-kumar) [reply MAINTAINING.md:157]: We may want to bucket issues differently depending on whether it is a clarifying question or if it is a proposal/discussion about a change. Let's keep them separate.

File: MAINTAINING.md (hunk 2 above)
Comment (harendra-kumar) [reply MAINTAINING.md:188]: ok.

File: MAINTAINING.md (hunk 3 above)
Comment (harendra-kumar) [reply MAINTAINING.md:207]: ok.

**Pull request merged**